lib/rototiller/task/params/env_var.rb

```diff
@@ -127,22 +107,22 @@ def reset
       end
 
       # @private
-      def check
+      def env_value_provided_by_user?
         # its possible that name could not be set
-        ENV.key?(@name) if @name
+        true if (ENV.key?(@name) if @name)
```

Comment:
An env var must have a name.  We raise an error if one is not provided.  
`raise(ArgumentError, 'A name must be supplied to add_env') unless @name`